server/controllers/linktaFlowController.ts

```diff
@@ -90,6 +93,10 @@ const createLinktaFlowController = (
         'Updating LinktaFlow for linktaFlowId:',
         req.params.linktaFlowId,
       );
+      // Check for authorization header. If not provided, throw 401 error
```

Comment:
The check for auth header in this controller is redundant here as it is already checked in the auth middleware.